.gitignore

```diff
@@ -0,0 +1,2 @@
+*~
+*.pyc
```

Comment:
Hi Stéphane,
The commit add gitignore at the repository level. it's not useful for the template. I should have done it in another PR. Thanks for the review !